samples/highcharts/blog/pie-chart-race/demo.details

```diff
@@ -0,0 +1,6 @@
+---
+ name: A pie chart race to display nuclear enegery production
```

Comment:
A bit too verbose for the title (this will show up in the demo thumbnail listing).  "Donut chart race" will do.